ts/plugins/linking/JolocomLinking.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  InteractionTransportType,
+  JolocomPlugin,
+  JolocomSDK,
+  TransportAPI,
+  TransportDesc,
+} from '@jolocom/sdk'
+import { Linking } from 'react-native'
+
+export enum LinkingErrorCode {
```

Comment:
Better to have a separate `ReactNativeJolocomError` (or smth like that) enum instead.

Reply:
It is available at the top level, as it is reexported from the index